### ISSUE TITLE
Perform a "Locate" from outside

### ISSUE THREAD

**Issue by virtusmankind** [3liz/lizmap-web-client#2374]
It would be nice if I can perform a "Locate by layer" using data come from external website. E.g.: I perform a query in a website with "City" and "Address" and the result of query is the input data in "Locate by layer" tool.

Linking an url from outside with right parameters, Lizmap (and QGis Server of course) would perform an "auto" locate by layer search within the map

I don't know if it is a Lizmap or QGis Server feature.

**Comment by Gustry** [comment]
Don't you want the search bar at the top ?
If you put google key, I think you can do geocoding etc, no ?

**Comment by virtusmankind** [comment]
No, I'd like to perform a "Locate by layer" search with feature data included in my project's layer (parcels, addresses, POI etc) grabbed outside Lizmap instead of directly write data in Locate by Layer box.
